sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -342,6 +342,8 @@ def upload_blob_from_url(self, source_url: str, **kwargs: Any) -> Dict[str, Any]
             operation will fail with ResourceExistsError.
         :keyword bool include_source_blob_properties:
             Indicates if properties from the source blob should be copied. Defaults to True.
+        :keyword dict(str, str) metadata:
+            Name-value pairs associated with the blob as metadata.
```

Comment:
FYI: This is what the type-hint looks like on every other API in this client:

```python
:param metadata:
            Name-value pairs associated with the blob as metadata.
        :type metadata: dict(str, str)
```
I added it as keyword if we want to make this non-breaking, but we can also make this breaking so that they are all params and just mark it as a bugfix

Also will make a note to add a changelog to this commit so we don't forget to document this

Reply:
Definitely don't make it breaking :). How about we make it a named keyword?